package.json

```diff
@@ -31,6 +31,7 @@
     "DOCUMENTATION.md"
   ],
   "devDependencies": {
+    "@chainlink/contracts": "0.6.1",
```

Comment:
Am I correct here? Probably this should not be a dev-dep, as long as postinstall needs to generate typings..

Reply:
Moved to `dependencies`.